### ISSUE TITLE
Argument list to long

### ISSUE THREAD

**Issue by Joly0** [kboghdady/youTube_ads_4_pi-hole#97]
Running the script gives the error "Argument list to long" and exits.
The same error is found here #50 and as i downloaded the script today, the fix is implemented, but it still gives me the error.

**Comment by Armag3ddon** [comment]
I get the same error, in line 57:
./youtube.sh: 57: ./youtube.sh: curl: Argument list too long

**Comment by Armag3ddon** [comment]
Trying to be helpful here:
I installed the script today. My PiHole has been running for quite some time by now (months). I don't know what this script is trying to achieve but sharedlogs is massive. For a test run, I redirected the output to bash and it filled my entire buffer. I don't think this is working as intended.

**Comment by faxesystem** [comment]
I simply disabled the last lines since the creation of the logs that are uploaded to the author causes this problem. 

`#### only disable if you don't like to share your youtube logs to be be added to my list
# sharedlogs=`sudo /usr/bin/sqlite3 /etc/pihole/pihole-FTL.db "select domain from queries where domain like '%googlevideo.com'" |uniq -d |tr '\n' ','`
# curl -sL "https://docs.google.com/forms/d/e/1FAIpQLSd_j3lQs_B7S3Hz3aA3IkwYMF4my0DnBMZFAn3e9grZo61VFQ/formResponse?usp=pp_url&entry.275594062=$sharedlogs"`

**Comment by Armag3ddon** [comment]
> I simply disabled the last two lines since the creation of the logs, that are uploaded to the author, seem to cause this problem. It even tells you to do that if you want. :)

Yes, you can do that but it completely defeats the purpose of sharing the blocked URLs so everyone can benefit from it.

**Comment by sonnyb9** [comment]
Disabling the last 2 lines stops the crowd sourced list of *googlevideo.com URLs, but if it doesn’t work anyway it’s the same result. I didn’t see anything the looks like a fix for this. Without a real fix, it’s self defeating.

**Comment by ohare93** [comment]
Agreed, this should be fixed for all to benefit
